README.md

```diff
@@ -51,10 +53,76 @@ If **RANGER_HOME** environment variable is not set, the default installation loc
 sudo bash ./scripts/deploy-ranger-service.sh
 ```
 
-## Execute Permission Check Service
+### Create/rename a GCS service on Ranger Admin App
 
-wip.
+The deployment script creates a service called "gcs".
+
+The name of the GCS service must match the name defined in ```ranger-gcs-security.xml```.
+You can find the setting under ```ranger.plugin.gcs.service.name``` property.
+
+The file is at ```/etc/gcs/conf/ranger-gcs-security.xml``` after deployment of the Ranger plugin proxy server
```

Comment:
Those are separated files. The server deployment script can't see the plugin's security setting since they are in different sub-project. The server deployment simply create a service called "gcs". The name is hard coded.
Alternatively, I can make the script and resources their own project so that they can view each other's value.